protocol/types.go

```diff
@@ -1,11 +1,11 @@
 package protocol
 
 import (
-	"github.com/mailru/easyjson"
+	"encoding/json"
 )
 
-// Note: all the types defined inside of this file will get the easyjson
-// interface contract be generated by `make generate-easyjson`
+// Note: all the types defined inside of this file will get the json
+// interface contract be generated by `make generate-json`
```

Comment:
This comment should be dropped, we don't have a generation target anymore.

Reply:
thanks for catching this